packages/block-editor/src/components/block-preview/index.js

```diff
@@ -113,7 +125,11 @@ export function useBlockPreview( { blocks, props = {}, layout } ) {
 		[]
 	);
 	const settings = useMemo(
-		() => ( { ...originalSettings, __unstableIsPreviewMode: true } ),
+		() => ( {
+			...originalSettings,
+			styles: undefined,
```

Comment:
To make sure I understand what's happening here, are the styles being set to undefined in this settings object the parent styles? Or the default children styles?

Reply:
This is clearing out the styles from the parent, so that the child starts off with a blank slate when it comes to the styles within the preview. I'll add an inline comment for this line so that it's clearer what's happening here.

Reply:
I've added a couple of comments in https://github.com/WordPress/gutenberg/pull/55288/commits/f2ca82166d3e343403ab9a7436005801a6c238f9 to hopefully clarify this a little better 🙂

Reply:
Nice, thanks!